ci/workflows/main.yml

```diff
@@ -0,0 +1,103 @@
+steps:
+  - label: "compute node build dev"
+    command: "ci/scripts/compute-node-build.sh -t debug -p dev"
+    key: "dev"
+    plugins:
+      - docker-compose#v3.9.0:
+          run: rw-build-env
+          config: ci/docker-compose.yml
+      - artifacts#v1.5.0:
+          upload:
+            - from: target/debug/risingwave
+              to: risingwave-dev
+            - from: target/debug/risingwave_regress_test
+              to: risingwave_regress_test-dev
+            - from: target/debug/risedev-playground
+              to: risedev-playground-dev
+    timeout_in_minutes: 10
+
+  - label: "compute node build release"
+    command: "ci/scripts/compute-node-build.sh -t release -p release"
+    if: build.source != "schedule"
```

Comment:
Better to have a separate `main-cron.yml`.

Reply:
If it is separate, there will be duplicate code, are you sure you want to do this?

Reply:
We can keep it as-is for now. But in the future, main cron will have far more jobs than PR or ordinary main builds.